system/hardware/tici/hardware.py

```diff
@@ -105,7 +105,7 @@ def bus(self):
   def nm(self):
     return self.bus.get_object(NM, '/org/freedesktop/NetworkManager')
 
-  @cached_property
+  @property
```

Comment:
this call only takes nanoseconds, caching is not important and causes issues when the modemmanager restarts
![image](https://github.com/commaai/openpilot/assets/9648890/0b01efd3-d364-4890-977c-3f26126ebc73)